packages/web-runtime/src/pages/accessDenied.vue

```diff
@@ -12,7 +12,8 @@
         </h3>
         <h4 v-translate class="oc-mb-m">Your user session is invalid or has expired.</h4>
         <div v-translate class="oc-mb-m" @click="performLogout">
-          If you like to login with a different user please proceed to <a id="exitAnchor">exit</a>.
+          If you like to login with a different user please proceed to
+          <router-link id="exitAnchor" :to="{ name: 'login' }">exit</router-link>
```

Comment:
The click on the router-link now does two things at the same time, and both are needed: the click handler on the outer div fires and the router link is being followed. This is a somewhat confusing mix.

I'd like to suggest a solution as follows:
1. remove the click handler from the outer div and remove the method from the script block
2. introduce a new page in `packages/web-runtime/src/pages` called `logout`
3. define a route in `packages/web-runtime/src/router/index.ts` called `logout` which renders the page component from (2)
4. in the script block of `logout.vue` the `created()` hook should immediately call the `logoutUser` method on the `authService`.

IMO that would be cleaner because there would be an actual link to follow. Which is expected for an inline text element that is styled as a link.

The template of the page should never be visible, but you could add some simple message to it like `Please wait while you're being logged out`.

Reply:
While this solution would be cleaner regarding the flow in the business logic, it still doesn't cause a redirect to the `login` route. I think that this needs to be done in the `user unloaded` event handler in `authService.ts` of the web-runtime. When I tried that a while back it caused redirect loops like crazy when using the regular logout from the top bar user menu.

Reply:
Thank you for your nice feedback
let us walk on this.